### DISCUSSION TITLE
Fingerprint ließt Fingerabdrücke nicht

### DISCUSSION THREAD

**Discussion by PinkyFooFoo** [frickelzeugs/FingerprintDoorbell#35] [Q&A]
Tag zusammen,

ich bin mittlerweile dabei dieses Projekt ein zweites Mal umzusetzten und habe den zweiten Sensor gerade angelernt. Hier begegnet mir offenbar gerade das Problem, dass der Sensor keine Fingerabdrücke ließt. 
-> Mircocontroller erkennt den Sensor, pairing ist erfolgreich
-> Enrollment startet erfolgreich, sensor blinkt pink
-> Sensor erkennt keinen Finger wenn er aufgelegt wird

Ich habe das ganze mittlerweile mehrfach neu verdrahtet und kann einen Fehler hier ausschließen. Alles doppelt und dreifach überprüft. Entweder ist beim Löten der Microcontroller kaputtgegangen, was ich mir nicht vorstellen kann oder der Sensor ist nicht in Ordnung. Daher wollte ich mal Fragen ob jemand von euch ähnliches schon erlebt hat?

Wie ist die Qualität der Sensoren bei euch? Kommt ihr zurecht oder sind häufiger Blindgänger dabei? Habe ich ggf. etwas übersehen?
-> Ich tausche jetzt mal den Sensor, wenn das nichts bringt den Microcontroller.

**Comment by frickelzeugs** [comment]
Wenn das Pairing funktioniert und auch der LED Ring fleißig in verschiedenen Farben blinkt, dann funktioniert die Serielle Kommunikation und würde einen Verdrahtungsfehler oder Microcontroller Defekt eher ausschließen. Vielleicht hat der Sensor ja wirklich einen Defekt, versuche Mal den anderen. Blinkt der Sensor rot bei Berührung? Ich habe 3 und im Bekanntenkreis nochmal 2 Sensoren und bislang keine derartigen Probleme beobachtet.

**Reply by PinkyFooFoo** [reply]
Ja, der Ring leuchtet - beim starten des Microcontrollers blinkt er fleißig blau, danach wechselt er ins breathing. 
Bei Berührung wechselt er sofort ins rote blinken. 
Wenn ich das Enrollment starte wechselt er sofort in pink blinkend. Dann SOLLTE er beim erkennen eines Abdrucks pink durchgehend leuchten - das passiert nicht. 

Der andere Scanner ist mittlerweile fest verbaut und nicht in Reichweite sodass ich "mal eben" austauschen könnte. Daher gehe ich den Weg über einen weiteren - den ich aber erst bestellt habe. Braucht ja immer ein paar Wochen bis das Zeug hier ist. Bietet mir Zeit zum debuggen/basteln am defekten.

**Reply by PinkyFooFoo** [reply]
Follow-Up: Hier war tatsächlich der Fingerabdruck-Scanner kaputt. Ein neuer hat sofort funktioniert. Wenn man die Discussions quer ließt dann fällt auch auf, dass die dinger häufiger mal "Kaputt" aus dem Werk kommen. Solle man beim Debuggen also ggf. beachten.
